### ISSUE TITLE
JS error "Error: routePathProperty is undefined TypeError: "routePathProperty is undefined""

### ISSUE THREAD

**Issue by valVk** [sulu/SuluArticleBundle#348]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| SuluArticleBundle Version | 0.7.1
| Sulu Version | 1.6.12

#### Actual Behavior

![selection_20180904_2671935](https://user-images.githubusercontent.com/1130626/45039795-0b299b00-b06d-11e8-9c2a-5624d5e5dfc5.png)

#### Expected Behavior

Save should be successful

#### Steps to Reproduce

Follow installation steps

**Comment by trickreich** [comment]
Sorry for the late feedback.
Sounds like you don't have a required property for the route.

Have a look in the example:
https://github.com/sulu/SuluArticleBundle/blob/develop/Resources/doc/article_default.xml#L40

**Comment by valVk** [comment]
Yes, I do not have that property because copied structure from another template(

Thanks.
